pkg/Directory.Build.props

```diff
@@ -6,6 +6,7 @@
          For offline builds we still set OfficialBuildId but we need to build all the packages for a single
          leg only, so we also take DotNetBuildFromSource  into account. -->
     <BuildingAnOfficialBuildLeg Condition="'$(BuildingAnOfficialBuildLeg)' == '' AND '$(OfficialBuildId)' != '' AND '$(DotNetBuildFromSource)' != 'true'">true</BuildingAnOfficialBuildLeg>
+    <HarvestStablePackage>false</HarvestStablePackage>
```

Comment:
Why is this now needed?

Reply:
Without it we'll try to harvest the stable version for these packages, however none of them have anything which we wish to harvest from.  We could include their previous version in the set of things we restore here https://github.com/dotnet/corefx/blob/07262aa6bce06f1e270e6a9b494bfe6eb1260b08/eng/restore/harvestPackages/harvestPackages.depproj#L13-L42 but its less work (and bandwidth) to just disable harvesting.